address-app/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.4/apache-maven-3.8.4-bin.zip
```

Comment:
Please add .gitignore in address-app folder and add .mvn folder also to prevent it from being pushed to the codeline. We only want to push source code files. The .mvn folder can be generated again from pom.xml locally.